acceptance/setup/pre_suite/40_install_deps.rb

```diff
@@ -1,93 +1,95 @@
-step "Install other dependencies on database" do
-  databases.each do |database|
-    os = test_config[:os_families][database.name]
-    db_facts = facts(database.name)
+unless (test_config[:skip_install])
+  step "Install other dependencies on database" do
+    databases.each do |database|
+      os = test_config[:os_families][database.name]
+      db_facts = facts(database.name)
 
-    use_our_jdk = ((db_facts["osfamily"] == "Debian") and
-                   (db_facts["operatingsystemmajrelease"] == "6" or
-                    db_facts["operatingsystemrelease"] == "10.04"))
+      use_our_jdk = ((db_facts["osfamily"] == "Debian") and
+                     (db_facts["operatingsystemmajrelease"] == "6" or
+                      db_facts["operatingsystemrelease"] == "10.04"))
 
-    # Install our JDK repository with a JDK 7 for Debian 6 and Ubuntu 10.04
-    # and install the oracle jdk
-    if use_our_jdk then
-      create_remote_file database, '/etc/apt/sources.list.d/jpkg.list', <<-REPO
-# Oracle JDK Packages
-deb http://s3-us-west-2.amazonaws.com/puppetdb-jdk/jpkg/ pljdk main
-      REPO
-      # Import GPG key
-      on database, "gpg --keyserver keys.gnupg.net --recv-keys B8615A77BBBFA17C"
-      on database, "gpg -a --export B8615A77BBBFA17C | apt-key add -"
-      on database, "apt-get update"
-    end
+      # Install our JDK repository with a JDK 7 for Debian 6 and Ubuntu 10.04
+      # and install the oracle jdk
+      if use_our_jdk then
+        create_remote_file database, '/etc/apt/sources.list.d/jpkg.list', <<-REPO
+  # Oracle JDK Packages
+  deb http://s3-us-west-2.amazonaws.com/puppetdb-jdk/jpkg/ pljdk main
+        REPO
+        # Import GPG key
+        on database, "gpg --keyserver keys.gnupg.net --recv-keys B8615A77BBBFA17C"
+        on database, "gpg -a --export B8615A77BBBFA17C | apt-key add -"
+        on database, "apt-get update"
+      end
```

Comment:
It's not whitespace-only; the whole thing is wrapped in a conditional, so it's indented in an extra two spaces.